pkg/controller/garbagecollector/config/types.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
2019 now ;)

Reply:
sorry, forgot it
will fix all comments tonight